test/passes/fpcast-emu_pass-arg=max-func-params@5.wast

```diff
@@ -0,0 +1,80 @@
+(module
+  (type $vijfd (func (param i32) (param i64) (param f32) (param f64)))
+  (type $jii (func (param i32) (param i32) (result i64)))
+  (type $fjj (func (param i64) (param i64) (result f32)))
+  (type $dff (func (param f32) (param f32) (result f64)))
+  (type $idd (func (param f64) (param f64) (result i32)))
+  (import "env" "imported_func" (func $imported-func (param i32 i64 f32 f64) (result f32)))
+  (table 10 10 funcref)
+  (elem (i32.const 0) $a $b $c $d $e $e $imported-func)
```

Comment:
this test could be much shorter and more focused I think. How about having just a module with a table and a single function in that table? Then we'd see it's instrumented to have five params, and that would show it works.

Reply:
Sounds reasonable. Should I change the original fpcast-emu test too?